packages/custom_lint/lib/src/analyzer_plugin/lints.dart

```diff
@@ -104,11 +105,18 @@ final _pluginNotStartedLintProvider = Provider.autoDispose
   return errors;
 });
 
+final invalidateLintsProvider =
+    Provider.autoDispose((ref) => Random().nextDouble());
+
 /// The list of lints per Dart Library emitted by a plugin, including
 /// built-in lints such as whether the plugin as started or not.
 final lintsForPluginProvider = StreamProvider.autoDispose
     .family<Map<String, plugin.AnalysisErrorsParams>, Uri>(
         (ref, linkKey) async* {
+  ref.watch(invalidateLintsProvider);
```

Comment:
@rrousselGit Any idea why invalidating `invalidateLintsProvider` doesn't cause this provider to enter a loading state? I'm also manually setting the state to loading just below here, but I can't seem to invalidate lints, but I'd like lints to be reported again after a hot reload to show that something is actually happening.